i18n/es.jsonp.js

```diff
@@ -241,17 +242,17 @@ document.localeJson = {
+      "about-contentinjection-para1": "Dependiendo de su navegador o framework, esta aplicación puede ser capaz de ejecutarse en diferentes modos, que llamamos \"Modo ServiceWorker\" y \"Modo Seguro\". Hay una opción en Configuración que le permite seleccionar entre estos modos. A continuación damos una explicación técnica de lo que hacen estos modos:",
+      "about-contentinjection-para2": "<b>Modo ServiceWorker</b>: Este es el modo predeterminado. Como su nombre indica, requiere que el navegador o el framework sea capaz de instalar un Service Worker, lo cual suele ser el caso en los navegadores modernos. Funciona interceptando las llamadas Fetch (solicitudes de red) del navegador o del framework y suministrando el contenido solicitado desde el ZIM. En este modo, el contenido se lee y se suministra tal cual desde el archivo al navegador. El contenido dinámico (por ejemplo, JavaScript) y las interfaces de usuario propietarias son totalmente compatibles con este modo. Este modo puede parecer inicialmente un poco más lento que el modo Seguro hasta que se almacenen en caché los activos de uso común, pero pronto iguala al modo Seguro en velocidad, al menos en los navegadores modernos. Sin embargo, los navegadores antiguos como IE11 son incompatibles con este modo, y la aplicación debe estar funcionando en un contexto Seguro (<code>https:</code>, <code>localhost</code>, o ciertas extensiones de navegador). Aunque este modo no es compatible de forma nativa con las extensiones de Mozilla (Firefox), proporcionamos una solución funcional al volver a lanzar la extensión como una Progressive Web App (PWA). Tenga en cuenta que este modo no se puede ejecutar con el protocolo <code>file:</code> (pero sólo IE11 y el antiguo Edge permiten que la aplicación se ejecute al lanzar <code>index.html</code> desde el sistema de archivos).",
+      "about-contentinjection-para3": "<b>Modo ServiceWorkerLocal</b>: Este modo sólo está disponible en las extensiones de Chromium. Es lo mismo que arriba, pero debido a las restricciones de seguridad muy estrictas en estas extensiones, no se puede ejecutar ningún JavaScript en línea o declaraciones eval. Desafortunadamente, los ZIM modernos pueden tener tal contenido dinámico, y por lo tanto la funcionalidad puede estar restringida en este modo. Sin embargo, es más Seguro: el contenido está en un sandbox y no se requiere ningún acceso al servidor para habilitar el Service Worker. Está protegido de la ejecución de código en línea a costa de la pérdida de algunas características en los ZIM dinámicos.",
```

Comment:
Ditto, "seguro" here is just an adjective meaning "secure", so don't capitalize.